msal-python-conceptual/index.md

```diff
@@ -74,6 +85,15 @@ Acquiring tokens with MSAL Python follows a three-step pattern. There will be so
        print(result.get("correlation_id"))  # You may need this when reporting a bug
    ```
 
+4. Save the code into a Python file locally, such as `msaltest.py`.
```

Comment:
Food for thought. Previously (and perhaps generally), the code snippets in a doc are for illustration purpose. They may not be runnable, at least there is no tooling to guarantee that.

Now that we added this step 4 & 5 which imply that the code snippet shall be runnable, shall we also leave a link to a self-contained, runnable, tested, [sample like this](https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/1.22.0/sample/interactive_sample.py)?